dmotion:
  com_y:
    data: [0, 0.57836361946367987, 1.3109177239701939, 2.7280000000000126, 3.6671817632569597, 4.8699999999999868, 5.5, -7.0009999999999986, -1.4341180724352032, 1.7452650158386773, 1.6100000000000001, 1.5155919777494309, 0.41000000000000025, -7.009999999999998, 0, 6.3858664527990685, 0, -0.29431600532895646, -0.29242046409558664, -2.2147544978134568, 0]